tools/telemetry-generator/src/handlers/appInsightsExecutionTimeTestHandler.ts

```diff
@@ -14,12 +14,17 @@ module.exports = function handler(fileData, telemetryClient: TelemetryClient): v
 	for (const testData of fileData.benchmarks) {
 		const arithmeticMeanMetricName = `${fileData.suiteName}_${testData.benchmarkName}_arithmeticMean`;
 		try {
+			const value = testData.customData["Period (ns/op)"];
+			if (isNaN(value)) {
+				console.error(`skipping metric ${arithmeticMeanMetricName} with value ${value} as it is not a number`);
```

Comment:
Particularly important here in case `value` is empty. Without the quotes, the message would just read very weird. Can you also apply it to the other instances of this?
```suggestion
				console.error(`skipping metric '${arithmeticMeanMetricName}' with value '${value}' as it is not a number`);
```

Reply:
Good idea. Applying it to all loggings